modules/pal/env/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../../../schemas/module-config.schema.json",
+  "name": "@pal/env",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+  },
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "node": "./package.json",
+      "types": "./src/index.d.ts",
+      "native": "./src/native/env.ts",
+      "web": "./src/web/env.ts",
+      "minigame": {
+        "default": "./src/minigame/env.ts",
+        "oppo": "./src/runtime/env.ts",
+        "vivo": "./src/runtime/env.ts",
+        "huawei": "./src/runtime/env.ts"
+      }
+    }
+  }
```

Comment:
here are the module conditional exports, which should replace the moduleOverride machnism in `cc.config.json`, see this schema design https://github.com/cocos/cocos-ccbuild/blob/f4d039f0bed5a5c753ce67d0f04c4fc8bafd130f/modules/modularize/src/module-config.ts#L53-L87